### DISCUSSION TITLE
How to resolve this issue? PWA offline doesn't respond

### DISCUSSION THREAD

**Discussion by KInGATiLLa** [facebook/docusaurus#4105] [Q&A]
![image](https://user-images.githubusercontent.com/17881842/105792578-a1561700-5fc2-11eb-831d-5fc67ea00835.png)

**Comment by slorber** [comment]
Hi,

Not 100% sure but it's likely a manifest issue. 

Maybe try:  `"start_url": "./index.html",` ?

**Reply by slorber** [reply]
It's likely a bug as the v2 Docusaurus site also has this issue:

Created https://github.com/facebook/docusaurus/issues/4108

Maybe try `offlineModeActivationStrategies: ["always"]` and report if it improves?

**Reply by KInGATiLLa** [reply]
Ok, I'll try to resolve. Thank you for your response
